docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+[View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundamental concept for the all components such as `Button`, `Image`, `Text`, and so on.
```

Comment:
@bshsqa 
... for the all components such as `Button`, `Image`, `Text`, and so on.
Usage of "the all" is not proper. 

Please use the rephrase mentioned in the previous comment, https://github.com/Samsung/tizen-docs/pull/962#discussion_r354233654.

Reply:
Thanks, I think I missed it :(